rclcpp/include/rclcpp/rclcpp.hpp

```diff
@@ -92,9 +92,11 @@
  *   - rclcpp::Event
  * - List topic names and types:
  *   - rclcpp::Node::get_topic_names_and_types()
- * - Get the number of publishers or subscribers on a topic:
+ * - Get the number of publishers, subscribers, clients, or services on a topic or service:
  *   - rclcpp::Node::count_publishers()
  *   - rclcpp::Node::count_subscribers()
+ *   - rclcpp::Node::count_clients()
+ *   - rclcpp::Node::count_services()
```

Comment:
I'm going to suggest that we write this slightly differently:

```suggestion
 * - Get the number of publishers or subscribers on a topic:
 *   - rclcpp::Node::count_publishers()
 *   - rclcpp::Node::count_subscribers()
 * - Get the number of clients or servers on a service:
 *   - rclcpp::Node::count_clients()
 *   - rclcpp::Node::count_services()
```

Reply:
Fixed in https://github.com/ros2/rclcpp/pull/2072/commits/2e5a93207cfbba53a8ccc425cdd0fb371dd5f207.